stackinator/builder.py

```diff
@@ -414,7 +414,7 @@ def generate(self, recipe):
 
         # Delete the store/repo path, if it already exists.
         # Do this so that incremental builds (though not officially supported) won't break if a repo is updated.
-        repo_dst = store_path / "repo"
+        repo_dst = store_path / pathlib.Path("spack_repo", "alps")
```

Comment:
Just wondering out of ignorance: is there a difference between what you wrote and
```suggestion
        repo_dst = store_path / "spack_repo" / "alps"
```
? I've been happily writing the latter, but I don't know if I'm missing some subtleties that make them different.